### PR TITLE
KPBMS-685: Add basic input component for BLKN project

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**

### CODE REVIEW

File: blkn/src/Input/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from "react";
+import Typography from "./../Typography";
+
+type Props = {
+  label?: string,
+  error?: string,
+  value?: string,
```
Comment (oreqizer) [inline blkn/src/Input/index.js]: I don't think `value` and `onChange` should be optional

File: blkn/src/Input/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  onChange?: (SyntethicInputEvent<HTMLInputElement>) => any,
+};
+
+const Input = (props: Props) => (
+  <div className="root">
```
Comment (oreqizer) [inline blkn/src/Input/index.js]: why a class if it is not used in the CSS?

File: blkn/src/Input/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <input value={props.value} onChange={props.onChange} />
+    </label>
+    {props.error && (
+      <div>
+        <Typography size="small" type="error">
+          {props.error}
+        </Typography>
+      </div>
+    )}
+    <style jsx>{`
```
Comment (oreqizer) [inline blkn/src/Input/index.js]: shouldn't some of the values be somehow centralized, such as `font-size`, colors, `line-height` etc?

File: blkn/src/Typography/index.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import * as React from "react";
+
+const getSize = size => {
```
Comment (oreqizer) [inline blkn/src/Typography/index.js]: wouldn't objects make more sense, especially if you require the `size` prop?

File: blkn/src/Typography/index.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      return "#171B1E";
+    case "error":
+      return "#D02228";
+    default:
+      return "#46515D";
+  }
+};
+
+type Props = {
+  size?: "large" | "small" | "normal",
```
Comment (oreqizer) [inline blkn/src/Typography/index.js]: a required prop with a default value makes more sense imo

File: blkn/src/Input/index.js (hunk 3 above)
Comment (jaroslav-kubicek) [reply blkn/src/Input/index.js]: that's definitely worth a thought, I'm just not decided yet

Comment (jakubzaba) [reply blkn/src/Input/index.js]: Will be optimized when it's needed - with more components.

**Pull request merged**